DependencyInjection/DoctrineMongoDBExtension.php

```diff
@@ -74,7 +74,7 @@ public function load(array $configs, ContainerBuilder $container)
             foreach ($config['resolve_target_documents'] as $name => $implementation) {
                 $def->addMethodCall('addResolveTargetDocument', [$name, $implementation, []]);
             }
-            $def->addTag('doctrine_mongodb.odm.event_listener', ['event' => 'loadClassMetadata']);
+            $def->addTag('doctrine_mongodb.odm.event_subscriber');
```

Comment:
This needs to be updated: we either require ODM 1.3 for version 3.6 of the bundle (which I'd like to avoid), or we check if the service is actually an event subscriber and add the appropriate tags.